parsl/configs/theta_local_htex_multinode.py

```diff
@@ -0,0 +1,30 @@
+from parsl.config import Config
+from parsl.providers import CobaltProvider
+from parsl.launchers import AprunLauncher
+from parsl.executors import HighThroughputExecutor
+from parsl.addresses import address_by_hostname
+
+
+config = Config(
+    executors=[
+        HighThroughputExecutor(
+            label='theta_local_htex_multinode',
+            max_workers=4,
+            address=address_by_hostname(),
+            provider=CobaltProvider(
+                queue="debug-flat-quad",
+                launcher=AprunLauncher(),
+                walltime="00:30:00",
+                nodes_per_block=2,
+                init_blocks=1,
+                max_blocks=1,
+                scheduler_options='',     # Input your scheduler_options if needed
```

Comment:
This isn't very informative, I favor a description of scheduler_options instead, for example:
```
# string to prepend to #SBATCH blocks in the submit script to the scheduler
```

Reply:
Agreed. I've updated all configs from my test group. Will update the rest once they are merged.